src/app/activity/bounty/Submit.ts

```diff
@@ -164,10 +165,18 @@ export const submitBountyMessage = async (submittedBounty: BountyCollection, sub
 	const claimantMessage: Message = await submittedByUser.send({ embeds: [embedMessage] });
 	await claimantMessage.react('🆘');
 
-	embedMessage.setFooter({text: '✅ - complete'});
+	let creatorReactionFooterText = '✅ - complete';
+	if (!submittedBounty.paidStatus || submittedBounty.paidStatus === PaidStatus.unpaid) {
+		creatorReactionFooterText = creatorReactionFooterText.concat(' | 💰 - mark as paid')
```

Comment:
Can all this paid message finagling that is repeated in all these activities be refactored into a single utility method? I think it would be cleaner.

Reply:
Yes - but if possible I'd like to table that to technical debt. I experimented around with it while writing this code, but wasn't able to come up with something clean that I liked.

The conditional logic with paid status is messy, but I don't think there's anyway around it.

Reply:
OK - Maybe we can clean it up as part of the List/Canonical card project, since that will touch a bunch of this